docs/content/preview/explore/change-data-capture/cdc-tutorials/cdc-confluent-cloud.md

```diff
@@ -14,3 +14,151 @@ type: docs
 ---
 
 ## Confluent Cloud
+
+This tutorial describes how to configure a YugabyteDB connector to publish changes to Confluent Cloud. Simultaneously, this assumes some familiarity with Docker and YAML files.
+
+![Architecture of YugabyteDB to Confluent Cloud pipeline](/images/explore/cdc/confluent_images/cdc_confluent_cloud.png)
+
+### Configuring Confluent Cloud cluster
+
+First, you will need to setup a Confluend Cloud cluster on the provider of your choice. For more information, visit [Manage Kafka Clusters on Confluent Cloud](https://docs.confluent.io/cloud/current/clusters/create-cluster.html).
+
+### Download the credentials
+
+Create a API key for your cluster and save it to be used further. See [Confluent Cloud API Keys](https://docs.confluent.io/cloud/current/access-management/authenticate/api-keys/api-keys.html) for reference.
+
+### Create custom Kafka Connect image with YugabyteDB connector
+
+To create a Kafka Connect image with the YugabyteDB connector, we start with the [base image](https://hub.docker.com/r/confluentinc/cp-server-connect/).
+
+1. Create a directory which will be used to store all related files.
+
+```sh
+mkdir kafka-connect-ccloud && cd kafka-connect-ccloud
+```
+
+<!-- TODO Vaibhav: Step 2 and 3 can be combined, see why is not working -->
+2. Download the YugabyteDB connector jar.
```

Comment:
number all steps 1 that way you don't have to keep track
indent code blocks and other content under steps so it will be indented correctly on final page